dev/com.ibm.ws.jsf.2.2_fat/test-applications/JSF22StatelessView.war/src/com/ibm/ws/jsf22/fat/statelessview/beans/faces40/StatelessViewBean.java

```diff
@@ -11,19 +11,21 @@
 /**
  * A simple managed bean that will be used to test
  * a view scoped managed bean.
- * 
+ *
  * @author Bill Lucy
  *
  */
 package com.ibm.ws.jsf22.fat.statelessview.beans.faces40;
 
 import java.io.Serializable;
 
-import javax.faces.bean.ManagedBean;
+import javax.enterprise.context.RequestScoped;
 import javax.faces.component.UIViewRoot;
 import javax.faces.context.FacesContext;
+import javax.inject.Named;
 
-@ManagedBean
+@Named
+@RequestScoped
```

Comment:
Default ManagedBean scope is `@RequestScoped` default CDI scope is `@Dependent` so just adding a scope here explicitly to ensure the test is the same for EE10.